.swiftlint.yml

```diff
@@ -6,7 +6,6 @@ analyzer_rules:
   - unused_declaration
 
 disabled_rules:
-  - trailing_whitespace
```

Comment:
Why was this rule disabled in the first place?  I generally hate picky linter settings, but this is one of the few rules I would want to have.

Reply:
likely because there were tons of places to be fixed